sale_stock_restocking_fee_invoicing/wizards/stock_return_picking.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockReturnPicking(models.TransientModel):
+
+    _inherit = "stock.return.picking"
+
+    charge_restocking_fee = fields.Boolean(
+        help="Tick this box if you wish to charge your customer a fee in "
+        "case of return of goods. Default value comes from the customer "
+        "info.",
+        default=False,
+    )
+    is_customer_return = fields.Boolean(readonly=True)
+
+    @api.model
+    def default_get(self, fields):  # pylint: disable=redefined-outer-name
+        res = super().default_get(fields)
+        if not (
+            "charge_restocking_fee" in fields
+            or "product_return_moves" in fields
+            or "is_customer_return" in fields
+        ):
+            return res
+        picking = self.env["stock.picking"].browse(self.env.context["active_id"])
+        charge_restocking_fee = picking.partner_id.charge_restocking_fee
+        if "is_customer_return" in fields:
+            res["is_customer_return"] = picking.location_dest_id.usage == "customer"
```

Comment:
Use the computed field you created
```suggestion
            res["is_customer_return"] = picking.is_customer_out
```